### PR TITLE
Add controlledRadioGroup

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/components/controlled/controlledInput/ControlledInput.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 type ControlledInputProps<T extends FieldValues> = UseControllerProps<T> &
   Omit<InputProps, 'value' | 'onChange'>
 
-export const ControlledInput = <T extends FieldValues>(props: ControlledInputProps<T>) => {
-  const { name, control, ...restInputProps } = props
-
+export const ControlledInput = <T extends FieldValues>({
+  name,
+  control,
+  ...restInputProps
```
Comment (vitalyabedik) [inline src/components/controlled/controlledInput/ControlledInput.tsx:13]: maybe better use naming ...restProps. Need to compare with ui components and do one similar variant

File: src/components/ui/radioGroup/RadioGroup.tsx [hunk 2]
```diff
@@ -6,7 +6,9 @@ import cn from 'classnames'
 import s from './RadioGroup.module.scss'
 import { RadioItem, RadioItemProps } from './radioItem'
 
-type RadioGroupProps = { options: RadioItemProps[] } & ComponentPropsWithoutRef<typeof Radio.Root>
+export type RadioGroupProps = { options: RadioItemProps[] } & ComponentPropsWithoutRef<
+  typeof Radio.Root
+>
 
 export const RadioGroup = forwardRef<ElementRef<typeof Radio.Root>, RadioGroupProps>(
   ({ options, className, ...props }, ref): JSX.Element => {
```
Comment (vitalyabedik) [inline src/components/ui/radioGroup/RadioGroup.tsx:14]: ...props or ...restProps or ...radioRestProps

Comment (VitaliKozhurov) [reply src/components/ui/radioGroup/RadioGroup.tsx]: Done!

File: src/components/controlled/controlledInput/ControlledInput.tsx (hunk 1 above)
Comment (VitaliKozhurov) [reply src/components/controlled/controlledInput/ControlledInput.tsx]: Done!

**Review by vitalyabedik — APPROVED** [review]
Good job!

**Review by DenisKulik — APPROVED** [review]

**Pull request merged**